flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/runtime/utils/BatchAbstractTestBase.java

```diff
@@ -47,7 +47,7 @@
 
 	private static Configuration getConfiguration() {
 		Configuration config = new Configuration();
-		config.setString(TaskManagerOptions.LEGACY_MANAGED_MEMORY_SIZE, "100m");
+		config.setString(TaskManagerOptions.LEGACY_MANAGED_MEMORY_SIZE, "200m");
```

Comment:
This change is needed because the some table batch tests have a pipelined region(corresponding to a slot sharing group) with managed memory sum to be more than 100m, so that these cases will pend at slot allocation. 
Previously they passed because it allocates shared slot with resources of the first arrived task slot request. This, however, make memory OOM possible to happen.

Reply:
Will move this change to an individual commit to describe the consideration.

Reply:
done.

Reply:
Forgot to update it that this fix is not needed now, since changes of FLINK-13708 are merged.
The issue in FLINK-13708 may result in large regions combined by vertices from different jobs. Solving it also make the existing tests able to pass with current managed memory setting.